doc/source/ray-air/examples/pytorch_tabular_batch_prediction.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import torch.nn as nn
+
+import ray
+from ray.data.preprocessors import Concatenator
+from ray.train.torch import to_air_checkpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
+
+
+def create_model(input_features):
+    return nn.Sequential(
+        nn.Linear(in_features=input_features, out_features=16),
+        nn.ReLU(),
+        nn.Linear(16, 16),
+        nn.ReLU(),
+        nn.Linear(16, 1),
+        nn.Sigmoid(),
+    )
+
+
+dataset = ray.data.read_csv("s3://anonymous@air-example-data/breast_cancer.csv")
+num_features = len(dataset.schema().names) - 1
+dataset = dataset.drop_columns(["target"])
```

Comment:
the `len() -1` is not needed if we dropped column here first ?

Reply:
done